File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// NasaOpenApis SDK utility: result_body

class NasaOpenApisResultBody
{
    public static function call(NasaOpenApisContext $ctx): ?NasaOpenApisResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/rb/utility/result_body.rb
# NasaOpenApis SDK utility: result_body
module NasaOpenApisUtilities
  ResultBody = ->(ctx) {
    response = ctx.response
    result = ctx.result
    if result && response && response.json_func && response.body
      result.body = response.json_func.call
    end
    result
  }
end
